server/modules/authentication/oidc/authentication.js

```diff
@@ -29,6 +29,15 @@ module.exports = {
               email: _.get(profile, '_json.' + conf.emailClaim)
             }
           })
+          const groups = _.get(profile, '_json.groups')
```

Comment:
Make the `groups` path a configurable property.

Reply:
I added a config entry for it :)